### PR TITLE
feat(web/applications): add representations page

### PR DESCRIPTION
**Pull request by jonathangoulding**
## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/server/applications/applications.router.js [hunk 1]
```diff
@@ -31,6 +32,9 @@ router.use('/create-new-case', guards.assertDomainTypeExists, applicationsCreate
 
 router.use('/case', guards.assertDomainTypeExists, applicationsCaseRouter);
 
+// router.use('/relevant-representations', guards.assertDomainTypeExists, relevantRepsRouter);
```
Comment (jonathangoulding) [inline apps/web/src/server/applications/applications.router.js:35]: do we need the domain assert ? clarity on what it does.

File: apps/web/src/server/views/applications/representations/representations.njk [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+			</strong>
+
+		</div>
+	</div>
+	<div class="govuk-grid-row">
+		<div class="govuk-grid-column-full">
+			{{ govukBreadcrumbs({
+				items: [
+					{
+						text: "Project documentation",
```
Comment (jonathangoulding) [inline apps/web/src/server/views/applications/representations/representations.njk:32]: back link does not work. 

Maybe a session guard issue.

File: apps/web/src/server/applications/case/documentation/applications-documentation.router.js [hunk 3]
```diff
@@ -24,9 +24,28 @@ applicationsDocumentationRouter
...
 
+import * as express from 'express';
+/**
+ *
+ * @param {express.Request} req
+ * @param {express.Response} res
+ * @param {Function} next
+ * @returns {*}
+ */
+function relevantRepsMiddleware(req, res, next) {
```
[the remark this answers is not in the document]
Comment (jonathangoulding) [reply apps/web/src/server/applications/case/documentation/applications-documentation.router.js:35]: Will be addressed in a cleanup ticket added - ASB-1398

File: apps/web/src/server/applications/applications.router.js [hunk 4]
```diff
@@ -31,6 +32,9 @@ router.use('/create-new-case', guards.assertDomainTypeExists, applicationsCreate
...
+router.use('/:id/relevant-representations', relevantRepsRouter);
```
[the remark this answers is not in the document]
Comment (jonathangoulding) [reply apps/web/src/server/applications/applications.router.js:36]: chanmged to prefix case

File: apps/web/src/server/views/applications/representations/representations.njk (hunk 2 above)
Comment (jonathangoulding) [reply apps/web/src/server/views/applications/representations/representations.njk:32]: Breadcrumbs work through dashboard journey.

File: apps/web/src/server/applications/applications.router.js (hunk 1 above)
Comment (jonathangoulding) [reply apps/web/src/server/applications/applications.router.js:35]: Implemented.

**Comment by francescobrenci-AR** [comment]
the representations folder should be moved inside the case folder

**Comment by jonathangoulding** [comment]
> the `representations` folder should be moved inside the `case` folder

done @francescobrenci-AR

File: apps/web/src/server/applications/case/representations/application-representations.view-model.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
+import { format } from 'date-fns';
```
Comment (lutamKC) [inline apps/web/src/server/applications/case/representations/application-representations.view-model.js:1]: some of these functions can be probably found in the nunjucks filters folder 'web/src/server/lib".'

For example I am sure the date formatting has been implemented already

**Review by lutamKC — APPROVED** [review]

**Pull request merged**